content/techniques/performance.md

```diff
@@ -74,6 +74,38 @@ You can pass options into the Fastify constructor through the `FastifyAdapter` c
 new FastifyAdapter({ logger: true });
 ```
 
+
+#### Middleware
+
+The middleware with `fastify` "works", but it retrieves only the raw `req` and `res` objects and not the fastify wrapper. This is a side effect of how `middie` works and `fastify` already talks about it in its documents for more information see [here](https://www.fastify.io/docs/latest/Reference/Middleware/).
```

Comment:
```suggestion
Middleware functions (when `@nestjs/platform-fastify` is used) retrieve the raw `req` and `res` objects instead of Fastify's wrappers. This is how the `middie` package works (that's used under the hood) and `fastify` - check out this [page(https://www.fastify.io/docs/latest/Reference/Middleware/) for more information,
```